### ISSUE TITLE
SPARQL processor update not importing 

### ISSUE THREAD

**Issue by alex-randles** [RDFLib/rdflib#1702]
Hi I have been using the following module for some time, however, it has randomly stopped working . The import statement is below. 

`from rdflib.plugins.sparql.processor import processUpdate`

I get the following import error.

```
Traceback (most recent call last):
  File "API.py", line 22, in <module>
    from modules.refinements import Refinements
  File "/home/alex/Desktop/Mapping-Quality-Framework/modules/refinements.py", line 7, in <module>
    from modules.validation_report import ValidationReport
  File "/home/alex/Desktop/Mapping-Quality-Framework/modules/validation_report.py", line 9, in <module>
    from rdflib.plugins.sparql.processor import processUpdate
  File "/home/alex/.local/lib/python3.6/site-packages/rdflib/plugins/sparql/__init__.py", line 33, in <module>
    from . import parser
  File "/home/alex/.local/lib/python3.6/site-packages/rdflib/plugins/sparql/parser.py", line 184, in <module>
    Param('prefix', PN_PREFIX)) + Suppress(':').leaveWhitespace()
  File "/home/alex/.local/lib/python3.6/site-packages/rdflib/plugins/sparql/parserutils.py", line 114, in __init__
    self.name = name
AttributeError: can't set attribute

```

**Comment by aucampia** [comment]
@alex-randles this problem was already fixed:

- https://github.com/RDFLib/rdflib/issues/1190
- https://github.com/RDFLib/rdflib/issues/1370
- https://github.com/RDFLib/rdflib/pull/1409

You can try to downgrade the version of pyparsing to 2.x or upgrade to the latest RDFLib, the fix should be included in RDFLib `6.1.1`.

**Comment by alex-randles** [comment]
Perfect its fixed, thanks for your help

I will close the issue
